### DISCUSSION TITLE
How to construct "not like" logic with graphql api?

### DISCUSSION THREAD

**Discussion by Stupidism** [payloadcms/payload#1294] [Q&A] [answered]
Currently, I only see these operators in the query filter.

<img width="910" alt="image" src="https://user-images.githubusercontent.com/5801015/197687694-bd77de56-af68-4fd2-91d5-24c8b3c48ca9.png">

I'm expecting sth like this:

```
slug: {
  not: {
    contains: "demo"
  }
}
```

<img width="874" alt="image" src="https://user-images.githubusercontent.com/5801015/197688044-23a5ebc9-8fbc-4a56-a3b7-562c548e7be0.png">

**Comment by jmikrut** [comment, marked as answer]
Hey @Stupidism — we don't currently have an operator that could represent `not like` but it could definitely be built. If you're up for making a PR, we would happily accept it!
